docs/zed/README.md

```diff
@@ -4,7 +4,7 @@
 
 > `zed` is a command-line tool to manage and query Zed data lakes.
 > You can import data from a variety of formats and `zed` will automatically
-> commit the data in the Zed data model's [super-structured](../formats/zed.md)
+> commit the data in the Zed data model's [super-structured](../formats/README.md)
```

Comment:
I suggested linking to this instead because the `zed.md` actually doesn't speak of "super-structured" at all, so it seems it would have had the potential to confuse.